magento/adobe-stock-integration
# #1494: Save image content hash to the media asset when saving image from the Adobe Stock

### Description (*)

This PR will implement the saving of image content hash to the media asset.
### Fixed Issues (if relevant)

1. Fixes https://github.com/magento/adobe-stock-integration/issues/1494: Save image content hash to the media asset when saving image from the Adobe Stock 
2. ...

### Manual testing scenarios (*)


1. Login to Admin panel
2. Open the media gallery. I.e. from the Catalog -> Category -> Content fieldset -> Select image from gallery
3. Click "Search Adobe Stock" button
4. Click on any image to open the image preview panel
5. Click "Save Preview" button
6. Confirm the image name in the popup

### Expected result (*)
The images information should be saved to the `media_gallery_asset` table. The hash filed should be populated with the `sha1` hash of the image content
